app/serializers/transcription_serializer.rb

```diff
@@ -6,4 +6,9 @@ class TranscriptionSerializer
     params[:serialize_text] == true
   }
   belongs_to :workflow
+
+  attribute :locked_by do |transcription|
+    transcription.locked_by if transcription.lock_timeout? &&
+                               DateTime.now < transcription.lock_timeout
```

Comment:
add a method to the model to help with this logic, perhaps
`transcription.in_lock_time_window?`
then in serializer you can 
`transcription.locked_by if transcription.in_lock_time_window?`

all the juicy bits of the implementation can be on the model, e.g. is there a timeout, is the timeout greater than the current time, etc.